lib/vendor/noitidart-ostypes/cutils.jsm

```diff
@@ -0,0 +1,462 @@
+/*** CTypes Utilities ***
+ * Stuff that I find useful in my CTypes work under all Operating Systems
```

Comment:
I'd like to see this as a dev-dep installed with npm. If they don't wanna accept a package.json, then I'd rather just have this checked in. Pretty much anything to avoid using two different package managers.